action.yml

```diff
@@ -46,29 +46,45 @@ runs:
 
           BASE_REF=${BASE_REF/refs\/heads\//}
           HEAD_REF=${HEAD_REF/refs\/heads\//}
-
-          echo "::set-output name=base_ref_branch::$(eval printf "%s" "$BASE_REF")"
-          echo "::set-output name=head_ref_branch::$(eval printf "%s" "$HEAD_REF")"
-
           REF_BRANCH=${REF/refs\/pull\//}
           REF_BRANCH=${REF_BRANCH/refs\/heads\//}
-
-          echo "::set-output name=ref_branch::$(eval printf "%s" "$REF_BRANCH")"
+          
+          if [[ -z "$GITHUB_OUTPUT" ]]; then
```

Comment:
@jackton1 Hi,

Thank you for keeping this Github action updated. I was actually looking at the code to see if this action would give me the warnings or if it has been updated to use the new syntax. I looked at the code and came across this. 

I was just curious, what does `[[ -z "$GITHUB_OUTPUT" ]]`do ? never seen this syntax before. 

Thanks!

Reply:
[[ -z "$GITHUB_OUTPUT" ]] is a test for whether the environment variable $GITHUB_OUTPUT exists